### ISSUE TITLE
Updates for new versions of SeleniumLibrary

### ISSUE THREAD

**Issue by huima** [boakley/robotframework-pageobjectlibrary#11]
Hi,

will you update the library to use newer versions of SeleniumLibrary ?

**Comment by boakley** [comment]
Yes, though it actually works now. You just need the Selenium2Library
wrapper (my library has a hard-coded reference to it), or you can subclass
PageObjectLibrary to change that hard-coded value.
